app/javascript/dashboard/routes/dashboard/search/components/SearchResultMessageItem.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <router-link :to="navigateTo" class="list-item">
+    <thumbnail :src="getThumbnail" size="42px" :username="getName" />
+    <div class="message-details">
+      <p class="name">{{ getName }}</p>
+      <div v-dompurify-html="messageContent" class="message-content" />
+    </div>
+  </router-link>
+</template>
+
+<script>
+import Thumbnail from 'dashboard/components/widgets/Thumbnail.vue';
+import messageFormatterMixin from 'shared/mixins/messageFormatterMixin';
+import { mapGetters } from 'vuex';
+import { frontendURL } from 'dashboard/helper/URLHelper.js';
+export default {
+  components: {
+    Thumbnail,
+  },
+  mixins: [messageFormatterMixin],
+  props: {
+    message: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  computed: {
+    messageContent() {
+      return this.formatMessage(this.message.content);
+    },
+    ...mapGetters({
+      accountId: 'getCurrentAccountId',
+    }),
+    navigateTo() {
+      return frontendURL(
+        `accounts/${this.accountId}/conversations/${this.message.conversation_id}`
+      );
+    },
+    getThumbnail() {
+      return this.message.sender && this.message.sender.thumbnail
```

Comment:
Adding these checks because, the `sender` is not sent in api requests for some messages.
CC: @tejaswinichile 
![Screenshot2022-12-13 at 22 47 47](https://user-images.githubusercontent.com/15716057/207404238-4edf3688-80fa-46df-9f85-6ba449952e39.png)